kitchen.dokken.yml

```diff
@@ -2,14 +2,13 @@
 driver:
   name: dokken
   privileged: true
-  chef_version: current
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
 
 transport:
   name: dokken
 
 provisioner:
   name: dokken
-  deprecations_as_errors: true
```

Comment:
It's probably worth leaving this so CI jobs fail if something new gets added that cookstyle can't catch

Reply:
Doesn't this get inherited from `kitchen.yml`?

Reply:
Oh yeah if you have it in the main config your GTG